conf/local.conf.sample

```diff
@@ -237,6 +237,11 @@ PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
 # this doesn't mean anything to you.
 CONF_VERSION = "1"
 
-# Remove nfs-utils which is being added from packagegroup-core-full-cmdline
-PACKAGE_EXCLUDE="nfs-utils"
-RDEPENDS_packagegroup-core-full-cmdline-sys-services_remove=" nfs-utils"
+PACKAGE_EXCLUDE="nfs-utils at tcp-wrappers"
+RDEPENDS_packagegroup-core-full-cmdline-sys-services_remove=" nfs-utils at tcp-wrappers"
+
+PACKAGE_EXCLUDE="ed"
```

Comment:
It seems it is not needed (all packages got removed). But it worth a change for the sake of syntax uniformity.